packages/components/select/__tests__/select.test.ts

```diff
@@ -393,6 +397,39 @@ describe('Select', () => {
     expect(findInnerInput().value).toBe('双皮奶')
   })
 
+  test('custom label', async () => {
+    wrapper = _mount(
+      `
+      <el-select v-model="value">
+        <el-option
+          v-for="item in options"
+          :label="item.name"
+          :key="item.id"
+          :value="item.id">
+        </el-option>
+      </el-select>
+    `,
+      () => ({
+        options: [
+          {
+            id: 1,
+            name: '黄金糕',
+          },
+          {
+            id: 2,
+            name: '双皮奶',
+          },
+        ],
+        value: {
```

Comment:
the `value` maybe should not a object
```suggestion
        value: 2
```

Reply:
you are right.

Reply:
the default value-key is `value`, I added another test